### DISCUSSION TITLE
Will the next Ceedling release 0.32.0 upgrade Unity to 2.6.0?

### DISCUSSION THREAD

**Discussion by lapo4719** [ThrowTheSwitch/Ceedling#867] [Q&A] [answered]
Will the next Ceedling release 0.32.0 upgrade Unity to 2.6.0? Or is there a path to upgrading the packaged Unity version in the future?

There are parametric test capabilities: TEST_CASE, TEST_RANGE, TEST_MATRIX that would be very useful

**Comment by mvandervoord** [comment, marked as answer]
:) Yes. It definitely will include the latest CMock and Unity.

**Reply by lapo4719** [reply]
Yay!

**Reply by lapo4719** [reply]
Is the next Ceedling release 0.32 (or 1.0.0!) going to be soon?

I'd love to use unity's  TEST_RANGE, TEST_MATRIX features, and I've put together a modest setup of unity in the meantime, but would love to come back to using w/ ceedling.

**Comment by jmrplens** [comment]
Hi @mvandervoord, in 0.32, what modifications have to be done in the project.yml for the TEST_CASE to work? I always get error in compilation if I define TEST_CASE and use it in some test, if I remove it it works correctly. In 0.31.1 works properly. thank you very much for your work! <3

> `EXCEPTION: ShellExecutionException ==> 'Default Test Compiler' (gcc.exe) terminated with exit code [1]
`

**Reply by mkarlesky** [reply]
Sorry, @jmrplens. That is probably a bug. I will look into it.

Incidentally, in the gigantic set of revisions for this next release, the Unity macros you are using are not compatible with Ceedling's vastly improved preprocessing. This will be restored in a future release.
